local-setup/jupyter/dataloader/unified_loader.py

```
@@ -3667,6 +3941,60 @@ def create_employees(self, employee_list: List[Dict], tenant: str,
+                                    "Employees": [existing_employee]
+                                }
+
+                                update_response = requests.post(update_url, json=update_payload, headers=headers)
+                                update_response.raise_for_status()
+
+                                results['password_updated'] += 1
+                                print(f"   [✓] [{i}/{len(employee_list)}] {emp_code} - Password updated for existing employee")
+                            else:
+                                print(f"   [⚠] [{i}/{len(employee_list)}] {emp_code} - Could not fetch existing employee for password update")
+
+                        except Exception as pwd_error:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)